src/saving/SaveHelper.cs

```diff
@@ -320,14 +320,16 @@ private static void PerformSave(InProgressSave inProgress, Save save)
     /// </summary>
     private static void QueueRemoveExcessAutoSaves()
     {
-        TaskExecutor.Instance.AddTask(new Task(() => DeleteExcessSaves("auto_save", Settings.Instance.MaxAutoSaves)));
+        TaskExecutor.Instance.AddTask(new Task(() =>
+            DeleteExcessSaves("auto_save", Settings.Instance.MaxAutoSaves)));
     }
 
     /// <summary>
     ///   Runs a background task for removing excess quick saves
     /// </summary>
     private static void QueueRemoveExcessQuickSaves()
     {
-        TaskExecutor.Instance.AddTask(new Task(() => DeleteExcessSaves("quick_save", Settings.Instance.MaxQuickSaves)));
+        TaskExecutor.Instance.AddTask(new Task(() =>
+            DeleteExcessSaves("quick_save", Settings.Instance.MaxQuickSaves)));
```

Comment:
I think these changes are unnecessary. Probably caused by when .Value was used these were too long lines, however these are no longer too long, right?

Reply:
The top one is 1 character short of 120 columns, the bottom is still too many. Usually when they get moved like that it's from the ruby script doing it.. I'm not sure why the bottom one wasn't getting detected as too long before though. Not sure if I should bother moving.

Reply:
The, could you try at least reverting the upper change to remove that change from this PR?